libraries/disp/viewers/channelselectionview.cpp

```diff
@@ -455,6 +454,13 @@ void ChannelSelectionView::loadSettings()
     } else {
         move(pos);
     }
+//    QList<QScreen*> screensList = QGuiApplication::screens();
+//    if(screensList.isEmpty())
+//    {
+//        move(QPoint(100,100));
```

Comment:
remove commented code if no longer necessary

Reply:
This was part of the code that solved a warning which we added back to trace some error. I'll add it back in.